nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/NvaGraph.java

```diff
@@ -19,7 +19,13 @@ public record NvaGraph(Model model) implements GraphValidable {
 
   public static NvaGraph fromJsonLd(JsonNode content) {
     var model = ModelFactory.createDefaultModel();
-    loadDataIntoModel(model, stringToStream(content.toString()));
+    loadJsonLdIntoModel(model, stringToStream(content.toString()));
```

Comment:
A bit sad for some data to be wrapped in JSON, but hey "attributes" and "backward compatibility"